packages/vm/Cargo.toml

```diff
@@ -29,6 +29,7 @@ bench = false
 bytes = "1.4.0"     # need a higher version than the one required by Wasmer for the Bytes -> Vec<u8> implementation
 clru = "0.6.1"
 crc32fast = "1.3.2"
+bech32 = "0.9.1"
```

Comment:
Why is this a runtime dependency? We might want to disable the whole `cosmwasm_vm::testing` module for non-test builds. But I guess this is a different discussion.

Reply:
Yeah, makes sense, thanks